sdb/commands/filter.py

```diff
@@ -24,6 +24,22 @@
 
 
 class Filter(sdb.Command):
+    """
+    Return objects matching expression
+
+    EXAMPLES
+        Print addresses greater than or equal to 4
+
+            sdb> addr 0 1 2 3 4 5 6 | filter obj >= 4
+            (void *)0x4
+            (void *)0x5
+            (void *)0x6
+
+        Print only pool "jax"
+
+            sdb> spa | filter obj.spa_name == "jax" | member spa_name
+            (char [256])"jax"
```

Comment:
As noted previously, let's point out that this is ZFS specific by describing the example with `Find the SPA object of the ZFS pool named "jax" and print its 'spa_name'` or something along these lines.

In addition you can also re-use the `dmesg | filter | count` example from my other comment.